packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -514,6 +514,17 @@ const parseOpenAPIAuth = (
     ? parseOpenAPIV3Auth(doc as OpenAPIV3.Document | OpenAPIV31.Document, op)
     : parseOpenAPIV2Auth(doc as OpenAPIV2.Document, op)
 
+const parseOpenAPIUrl = (doc: OpenAPI.Document): string => {
+  let url = ""
+  if (objectHasProperty(doc, "host")) {
+    url += doc.host
+  }
+  if (objectHasProperty(doc, "basePath")) {
+    url += doc.basePath
+  }
+  return url
+}
+
```

Comment:
There are differences to how OpenAPI v3 and OpenAPI v2 (Swagger) deals with the URL stuff.

Please see the differences here:
- Relevant v3 reference: https://swagger.io/specification/#server-object
- Relevant v2 reference: https://swagger.io/specification/v2/#:~:text=to%20be%20obscured.-,Schema,-Swagger%20Object

This implementation only seems to be handling the V2 case and then blanking out the base URL part in v3.